packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -333,40 +333,38 @@ fn batch_to_js<'m>(
 ) -> Result<mv8::Value<'m>> {
     // TODO: Is `mem.data.len()` different from `mem.size`? (like Vec capacity vs len?)
     let arraybuffer = mv8.create_arraybuffer(mem.data.as_ptr(), mem.size);
-    let batch_id = mem.get_id();
+    let batch_id = mem.id();
```

Comment:
```suggestion
    let mem_id = mem.id();
```

Might be worth looking for `batch_id` all over the place just in case I've missed some.

Reply:
Are they though? I thought we were rethinking things to say that batches don't have id's? Batches just happen to be in memory segments which are allocated by an id. (I was actually going to suggest we rename id to something clearer like shmem_os_id or whatever)